packages/react/src/components/Code/index.tsx

```diff
@@ -64,17 +79,33 @@ const Code: React.ComponentType<React.PropsWithChildren<Props>> = ({
   }, [scrollable]);
 
   return (
-    <Highlighter
-      {...props}
-      PreTag={PreWithRef}
-      useInlineStyles={false}
-      className={classNames('Code', className, {
-        'Code--scrollable': scrollable
-      })}
-      tabIndex={scrollableRegion ? 0 : undefined}
-    >
-      {children}
-    </Highlighter>
+    <>
+      {(label || allowCopy) && (
+        <div className="CodeWrapper">
+          {label && <div id="code-label">{label}</div>}
```

Comment:
The id here should be generated, not static.

There's an example of how to implement this on Combobox with `useId`: https://github.com/dequelabs/cauldron/blob/f1f016388543360dc6a525037d583dc3b8f72835/packages/react/src/components/Combobox/Combobox.tsx#L420-L427

Reply:
Fixed, create dynamic id for label